src/components/filesystem/index.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
             normalSortOrder: true,
             clipboard:{},
             selectedFiles:[],
+            captureChunks:[],
```

Comment:
how big can this get?

Reply:
each chunk is 5 seconds worth of data. The array decreases in size after each chunk is written to storage.